### PR TITLE
Minor UI changes

### PR DESCRIPTION
**Pull request by domanttas**
Added some logic and displaying messages based on whether person was found. Need styling and some images/actions with them, so app would look alive. (Did not fix changes from last PR, as I don't have much time right now, wanted quick finishing up).

### CODE REVIEW

**Comment by domanttas** [comment]
Thinking of changing checking with strings to some kind of ENUM, it's a TODO.

**Comment by domanttas** [comment]
I'm aware it's not a good practise :D

File: WhoIsThat/WhoIsThat/ViewModels/HomeViewModel.cs [hunk 1]
```diff
@@ -71,5 +92,15 @@ protected virtual void OnPropertyChanged(string propertyName)
                 PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
             }
         }
+
+        protected bool IsIdentified(string message)
+        {
+            if (message == "No faces were detected!" || message == "No one was indetified!")
+            {
```
Comment (tomasmuzas) [inline WhoIsThat/WhoIsThat/ViewModels/HomeViewModel.cs]: I really suggest having some constants class

**Comment by domanttas** [comment]
@tomasmuzas is this appropriate?

**Pull request merged**